src/index.js

```diff
@@ -1,9 +1,13 @@
 import './style.css';
-// eslint-disable-next-line import/no-named-default
-import { default as checkboxesEvent, list } from './status-update.js';
-// eslint-disable-next-line import/no-named-default
```

Comment:
- [ ] please visit my comment above for disabling eslint rules and apply it to everywhere you disabled these rules